dbfit-java/build.gradle

```diff
@@ -222,3 +251,114 @@ project('netezza') {
     }
 }
 
+// Aggregated dependencies to include in the POM file when publishing.
+dependencies {
+    compile publicCoreCompileDeps
+    runtime publicHsqldbRuntimeDeps
+    compile publicOracleCompileDeps
+    compile publicMysqlCompileDeps
+    compile publicPostgresCompileDeps
+}
+
+// Remove the default artifact.
```

Comment:
The default artefact (artifact) for this project is `dbfit-java-X.Y.Z.jar` which has no class files in it as there are no Java source files in this project. 

The artefact would get signed and published, by default, along with the additional aggregate JARs we create. I can enhance the comments if needed.